### ISSUE TITLE
Make ClusterControl run in unprivileged container (not root)

### ISSUE THREAD

**Issue by akram** [severalnines/docker#22]
Hi,

it would be great if we could have an image that does not require root privileged. Especially to make it run on OpenShift which by default runs image in "restricted" scc.

This would for sure require to use an SSH daemon that does not use privileged ports. Or maybe, trying to replace SSH with an alternative tool.

**Comment by pantaoran** [comment]
Yes please do this. As I'm investigating ClusterControl, this is an immediate blocker. I cannot use your product if it is running as root.
